themes/vocabulary_theme/templates/blog-post.html

```diff
@@ -6,55 +6,70 @@
 
-{% block title %}{{ this.title }}{% endblock %}
-{% block header %}{{ this.parent.title }}{% endblock %}
 {% block body %}
-  <h2 class="mb-0">{{ this.title }}</h2>
-  {{ check_file('content' + this.parent.parent.path + '/authors/' + this.author + '/contents.lr') }}
-  <p class="meta text-muted mt-0 mb-2">by <a href="{{ author|url }}">{{ render_author_name(author) }}</a>
-  on {{ this.pub_date|dateformat('EEEE, yyyy-''MM-d') }}</p>
-  {% if this.series|length %}  
-   {% set series = this.parent.parent.children.get('series').children.get(this.series) %}
-  <div class="card bg-light mb-2">
-    <div class="card-body p-2">
-    <p class="meta my-0"><span class="mr-2"><strong>This blog is part of the series:</strong></span>
-       <a href="#series">{{ series.name }}</a>
-    </p>
+  <div class="single-entry">
+    <div class="single-entry-header">
+      <div class="container">
+        <h4>CC Open Source Blog</h4>
+        <h2 class="title">{{ this.title }}</h2>
+        {{ check_file('content' + this.parent.parent.path + '/authors/' + this.author + '/contents.lr') }}
+        <div class="author columns is-marginless">
+          {% if author.md5_hashed_email %}
+            <figure class="image profile">
+              <img src="https://secure.gravatar.com/avatar/{{ author.md5_hashed_email }}?size=200" alt="gravatar" class="profile" />
+            </figure>
+          {% endif %}
+          <p class="column">by <a href="{{ author|url }}">{{ render_author_name(author) }}</a>
+          on {{ this.pub_date|dateformat('yyyy-''MM-d') }}</p>
+        </div>
+      </div>
     </div>
-  </div>
- {% endif %}
-  <div class="body">{{ this.body }}</div>
-  {% if this.series|length %}
-  <div class="mt-3 bg-light" id="series">
-    <hr/>
-    <h5 class="font-weight-bold">Posts in the <em>{{ series.name }}</em> series</h5>
-    <ul class="toc list-unstyled">
-      {% for post in series.children.order_by('pub_date') %}
-        <li>
-        {% if post.path==this.path %}
-          <strong>{{ post.title }}</strong>
-        {% else %}
-          <a href="{{ post|url }}">{{ post.title }}</a>
-        {% endif %}
-        </li>
-      {% endfor %}
-    </ul>
-    <hr/>
-  </div>
-  {% endif %}
-  <div class="card bg-light">
-    <div class="card-body p-2">
-      <p class="meta my-0"><span class="mr-2"><strong>Categories:</strong></span>
-        {% if this.categories|length %}
-          {% for category in this.categories %}
-            {% set current_category = this.parent.parent.children.get('categories').children.get(category) %}
-            {{ check_file('content' + this.parent.parent.path + '/categories/' + category + '/contents.lr') }}
-            <a href=" {{ current_category|url }} ">{{ category }}</a>{{ "," if not loop.last }}
-          {% endfor %}
-        {% else %}
-          none
-        {% endif %}
-      </p>
+    <div class="single-entry-body container">
```

Comment:
this needs more padding inside the content. the content is too close to the header 
![Screen Shot 2020-06-22 at 12 19 20 PM](https://user-images.githubusercontent.com/894708/85311008-be51c700-b482-11ea-9a1d-68df73294800.png)

Reply:
Regarding this, I am waiting for Francisco's response on mockups.